modules/mono/csharp_script.cpp

```diff
@@ -72,7 +72,7 @@
 
 // Types that will be skipped over (in favor of their base types) when setting up instance bindings.
 // This must be a superset of `ignored_types` in bindings_generator.cpp.
-const Vector<String> ignored_types = { "PhysicsServer2DExtension", "PhysicsServer3DExtension" };
+const Vector<String> ignored_types = {};
```

Comment:
I would hope we don't need this again, but it's nice to have because if a non-.NET PR introduces some class that breaks the .NET module, then we can just tell the contributor to add the class to the `ignored_types` instead of disabling CI.